fontMetricsData.js

```diff
@@ -1,6 +1,7 @@
 // This file is GENERATED by buildMetrics.sh. DO NOT MODIFY.
```

Comment:
Is this comment still accurate? If so, it seems we should be modifying that build process (perhaps `src/metrics/mapping.pl`) instead of/in addition to directly modifying this file...

Reply:
@edemaine This file is modified as a result of `extract_ttfs.py`, which is part of `buildMetrics.sh`.

Reply:
Indeed it is. Sorry, it's been a long time since touching the fonts. I confirmed that this is the `fontMetricsData.js` constructed by `buildMetrics.sh`.